tests/jax/core_system_components/parameter_server_test.py

```diff
@@ -58,13 +62,38 @@ def design(self) -> Tuple[DesignSpec, Dict]:
         return components, {}
 
 
+class TestParameterServer(HookOrderTracking, ParameterServer):
+    def __init__(
+        self,
+        config: SimpleNamespace,
+        components: List[Callback],
+    ) -> None:
+        """Initialise the parameter server."""
+        self.reset_hook_list()
+
+        super().__init__(config, components)
+
+
 @pytest.fixture
 def test_system() -> System:
     """Dummy system with zero components."""
     return TestSystem()
 
 
-def test_parameter_server(
+@pytest.fixture
+def test_parameter_server() -> ParameterServer:
+    """Dummy parameter server with no components"""
+    return TestParameterServer(
+        SimpleNamespace(
```

Comment:
Very minor comment, but I find it easier to read code when you explicitly have the keywords included. For example, 
`return TestParameterServer(
        config = SimpleNamespace(
            config_key="expected_value",
            non_blocking_sleep_seconds=1,
            get_parameters="parameter_list",
        ),
        components=[], 
    )`
This is quite nitpicky so please feel free to ignore.

Reply:
Agreed. Fixed 👍